src/idl_gen_ts.cpp

```diff
@@ -406,6 +406,21 @@ class TsGenerator : public BaseGenerator {
       // return the bigint value directly since typescript does not support
       // enums with bigint backing types.
       switch (value.type.base_type) {
+        case BASE_TYPE_ARRAY: {
+          std::string ret = "[";
+          for (auto i = 0; i < value.type.fixed_length; ++i) {
+            ret +=
+                AddImport(imports, *value.type.enum_def, *value.type.enum_def)
+                    .name +
```

Comment:
this is odd formatting, did you run the scripts/clang-format-git.sh command?

Reply:
I think I haven't run it last time. But it doesn't make any correction on this line when I re-run it. (Does mostly between 570-626 though) I will double check.

Reply:
clang-format-git.sh is happy now. c716af815b3b6b5dd3017090a5bdd0ca434d4470